src/installer/records.py

```diff
@@ -111,6 +123,22 @@ def __repr__(self):
             self.path, self.hash_, self.size,
         )
 
+    def __eq__(self, other):
+        # type: (object) -> bool
+        if not isinstance(other, Record):
+            return NotImplemented
```

Comment:
Did you mean `raise` here?

Reply:
No, this [should be returned](https://docs.python.org/3.9/library/constants.html#NotImplemented) AFAIK.

Reply:
No -- returning here is fine: https://docs.python.org/3/library/constants.html#NotImplemented

Reply:
Ah, sorry.